src/application/App.tsx

```diff
@@ -38,14 +36,13 @@ const GET_OPERATION_COUNTS: TypedDocumentNode<
 `;
 
 export const App = (): JSX.Element => {
+  const [open, setOpen] = useState(false);
```

Comment:
Maybe use a more descriptive name like `settingsOpen` here?

Reply:
Thanks for the reminder 🙂. I meant to come back and rename, then got distracted by everything else 🤣. Done [dc3ebcd](https://github.com/apollographql/apollo-client-devtools/pull/1180/commits/dc3ebcdb48329ab77b2d025e92111c7ebd51eb21)